### ISSUE TITLE
Missing Localization String in Admin Column Edit

### ISSUE THREAD

**Issue by fabiangigler** [codepress/admin-columns-issues#1254]
The labels "Edit" and "Clone" in the Admin Column Settings are not localized.

Screenshot of the Admin Screen (de_DE) :
![grafik](https://user-images.githubusercontent.com/3050232/60450612-8c6ca280-9c2a-11e9-971a-fdfd80dd8036.png)

The String for "Clone" is also not in transifex:
![grafik](https://user-images.githubusercontent.com/3050232/60450780-e5d4d180-9c2a-11e9-8264-690ea392e310.png)

**Comment by fabiangigler** [comment]
I just realized there are several String on the same page that are not Localized in the UI and not showing up in Transifex:
![grafik](https://user-images.githubusercontent.com/3050232/60450935-35b39880-9c2b-11e9-8063-91dbacde3e9b.png)

In the Tabs on top: "Settings"
Next to the Column Selector: "View"
On the Bottom of the Columns: "Add Column"


I can supply the german translations as soon as the strings are added in transifex.

**Comment by DGStefan** [comment]
@fabiangigler 
Thanks for your input.
I will check the plugin for the missing string.
Also be aware that our pro version contains the free version that has its own translation file managed on wordpress.org.
https://translate.wordpress.org/locale/de/default/wp-plugins/codepress-admin-columns/

**Comment by DGStefan** [comment]
@fabiangigler 
I checked and both the strings for 'Edit' and 'Clone' are part of the free version which can be translated on Wordpress.org. And the same goes for all the other string you mention.

**Comment by fabiangigler** [comment]
Hey  @DGStefan thanks for the answer, you are indeed right about that.
I just added some german translations.

However the string translations for "Clone" and "Edit" were already existing prior to my added translations. I also checked the plugin files and it seems like the strings are properly passed through the _e() translation function.

BUT I still see a weird mix of German and English in my WordPress Backend:
![grafik](https://user-images.githubusercontent.com/3050232/61879191-e0e60380-aef2-11e9-8160-c3c03607a5fb.png)
